specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/2019-10-28-preview/iotcentral.json

```diff
@@ -9,30 +9,18 @@
   "x-ms-parameterized-host": {
     "hostTemplate": "{subdomain}.{centralDnsSuffixInPath}",
     "parameters": [
-      {
-        "$ref": "#/parameters/CentralDnsSuffixInPath"
-      },
-      {
-        "$ref": "#/parameters/Subdomain"
-      }
+      { "$ref": "#/parameters/CentralDnsSuffixInPath" },
```

Comment:
The formatting of this doc is wrong, please see https://aka.ms/AA6h31t  on how to fix.

Reply:
Got it, thanks